semgrep-core/tests/OTHER/rules/taint_sanitizer_var.py

```diff
@@ -0,0 +1,11 @@
+def ok():
+    x = source
+    # `x` is sanitized by side effect!
+    sanitize(x)
```

Comment:
would be good to also add test examples that shows the limitations you mention in the changelog no longer applies here.
I guess the old solution was to do  pattern-inside: sanitize($X) AND pattern: $X 
and that was sanitizing every $X, so some code with some if and reassign to x later to source would
help show that the limitations no longer applies with focus-metavariable.

Reply:
Good point!